playground/src/input-bar/input-bar.scss

```diff
@@ -0,0 +1,57 @@
+@import 'base.scss';
```

Comment:
Noooo. That's the reason we've got the following in the playground CSS a dozen times:

https://github.com/ampproject/amp.dev/blob/38fd7ef3dba9ebc6b822b21308a0ec5f9282fa07/frontend/scss/base.scss#L7-L9

Already removed it for the other components in an upcoming PR.